addons/project/data/project_data.xml

```diff
@@ -50,7 +50,6 @@
             <field name="name">Task Rating</field>
             <field name="res_model">project.task</field>
             <field name="default" eval="True"/>
-            <field name="description">Ratings</field>
```

Comment:
Btw just a note, all of these are with "noupdate true", so it will require a migration (when reaching master) to remove existing occurrences. (we can wait forward port to do it)